src/boot/efi/device-path-util.c

```diff
@@ -3,6 +3,70 @@
 #include "device-path-util.h"
 #include "util.h"
 
+EFI_STATUS make_multiple_file_device_path(EFI_HANDLE device, const char16_t **files, EFI_DEVICE_PATH
+                                          ***ret_dp)
+{
```

Comment:
plesase follow our usual coding style, i.e. line break after the `(`, then indent to 16spaces, then put `{` on the same line as `(`